projects/samples/environments/factory/worlds/hall.wbt

```diff
@@ -29,38 +29,44 @@ Floor {
   translation -0.14 0 0
   size 22.54 6.95
   tileSize 2 2
-  appearance PBRAppearance {
-    baseColorMap ImageTexture {
-      url [
-        "textures/floor_base_color.jpg"
-      ]
-    }
-    roughnessMap ImageTexture {
-      url [
-        "textures/floor_roughness.jpg"
-      ]
-    }
-    metalness 0
+  appearance BakelitePlastic {
   }
 }
-Floor {
+Solid {
   translation -3.2 0 -4.985
-  name "floor(1)"
-  size 6.3 3.02
-  tileSize 2 2
-  appearance PBRAppearance {
-    baseColorMap ImageTexture {
-      url [
-        "textures/floor_base_color.jpg"
-      ]
-    }
-    roughnessMap ImageTexture {
-      url [
-        "textures/floor_roughness.jpg"
-      ]
+  children [
+    Shape {
+      appearance BakelitePlastic {
+      }
+      geometry IndexedFaceSet {
+        coord Coordinate {
+          point [
+            -3.15 0 -1.51
+            3.15 0 -1.51
+            -3.15 0 1.51
+            3.15 0 1.51
+          ]
+        }
+        texCoord TextureCoordinate {
+          point [
+            0 0
+            3.15 0
+            0 1.51
+            3.15 1.51
+          ]
+        }
+        coordIndex [
+          0, 2, 3, 1, -1
+        ]
+        texCoordIndex [
+          0, 2, 3, 1, -1
+        ]
+      }
     }
-    metalness 0
-  }
+  ]
+  name "floor(1)"
```

Comment:
```suggestion
  name "floor"
```

Reply:
That's not going to be possible as the `name` of the `Floor` PROTO is `floor` too, but actually what I can do is to transform this `Solid` into a `Transform` to get ride of this field.